### DISCUSSION TITLE
Unable to install via `npx shadcn@latest add calendar`

### DISCUSSION THREAD

**Discussion by nborsh** [gpbl/react-day-picker#2512] [Support] [answered]
Thanks for taking the time for writing this bug report.

## Code 

Please include the code required to reproduce the bug.

```tsx
import { DayPicker } from "react-day-picker";

export function Start() {
  return (
    <DayPicker  />
  );
}
```

## Expected Behavior

I want to add the calendar component by using "npx shadcn@latest add calendar", but I'm getting the following error. I've been able to add other components, like the card and table, but I'm getting an error with the calendar.

## Actual Behavior

I'm getting an error preventing me to adding the calendar. 

## Screenshots

<img width="879" alt="image" src="https://github.com/user-attachments/assets/cd566d9d-3093-48ee-ae1d-754f20015050">

**Comment by gpbl** [comment, marked as answer]
@nborsh it looks like that you need to install the previous version to make it working with `shadcn`:

```
peer date-fns@"^2.28.0 || ^3.0.0" from react-day-picker@8.10.1
npm error node_modules/ react-day-picker
npm error
react-day-picker@"8.10.1" from the root project
```

Try with a `pnpm install react-day-picker@8`...
